pre-commit/action.yaml

```diff
@@ -14,7 +14,7 @@ runs:
   steps:
     # shfmt requires Go 1.16+
     - name: Set up go
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@v4
       with:
         go-version: ^1.16
```

Comment:
I think we can remove the section now.
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#go
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#cached-tools

Reply:
Ah, but considering ARM self-hosted machine, keeping it as it is would be better.